distribution/build.gradle

```diff
@@ -40,6 +41,20 @@ buildscript {
 }
 
 /*****************************************************************************
+ *                                Notice file                                *
+ *****************************************************************************/
+
+// integ test zip only uses core, so a different notice file is needed there
+task buildCoreNotice(type: NoticeTask) {
+  dependencies project(':core')
+}
+
+// other distributions include notices from modules as well, which are added below later
+task buildFullNotice(type: NoticeTask) {
+  dependencies project(':core')
```

Comment:
Can you document why we need both? Or maybe we don't need both and this is a leftover?